cmds/boot/pxeboot/pxeboot.go

```diff
@@ -11,12 +11,12 @@
 // ServerName options (which may be embedded in the original BOOTP message, or
 // as option codes) to find something to boot.
 //
-// This BootFileName may point to
+// # This BootFileName may point to
```

Comment:
Is the # here needed?

Reply:
See [Go1.19 Release Notes - Doc Comments](https://go.dev/doc/go1.19). This line is interpreted as a heading under the new syntax. I will push a fix.